redis/3-highly-available/README.md

```diff
@@ -33,3 +33,158 @@ The following variables are supported
 Variable | Description | Expected values | Default
 --- | --- | --- | ----
 REDIS_MASTER_DOWN_AFTER_MILLISECONDS | How many milliseconds to wait before starting a master failover (if two or more redis sentinels agree) | integer (milliseconds) | 5000
+
+### Example Automatic Deployment via continuous-pipe.yml
+
+The following are some example tasks for automating the deployment of a highly available cluster:
+
+```yaml
+variables:
+  REDIS_REPLICAS: 3
+  REDIS_SENTINEL_REPLICAS: 3
+
+tasks:
+  infrastructure:
+    deploy:
+      services:
+        mysql: ~
+
+  redis_ha_master_deployment:
+    filter:
+      expression: 'tasks.infrastructure.services.mysql.created'
+    deploy:
+      services:
+        redis_initial_master:
+          specification:
+            source:
+              image: quay.io/continuouspipe/redis3-highly-available:stable
+            accessibility:
+              from_cluster: true
+            ports:
+              - 6379
+              - 26379
+            resources:
+              requests:
+                cpu: 50m
+                memory: 100Mi
+              limits:
+                cpu: 250m
+                memory: 500Mi
+            command:
+              - /bin/bash
+              - -c
+              - '/usr/local/bin/redis_command run_master_and_sentinel'
```

Comment:
Why don't we call it `container` as the rest?

Reply:
Done!